hack/build-multiarch-kubemacpool.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+ARCH=$1
+PLATFORMS=$2
+KUBEMACPOOL_IMAGE_TAGGED=$3
+KUBEMACPOOL_IMAGE_GIT_TAGGED=$4
+DOCKER_BUILDER=$5
+OCI_BIN=$6
+REGISTRY=$7
+
+if [ "$OCI_BIN" == "docker" ]; then
```

Comment:
Please switch order, as we encourage to use podman all over

Reply:
I don’t think the order will make a difference here since we’re passing the OCI_BIN value from the Makefile, and the Makefile already takes care of it. You can see the relevant part of the Makefile here: [link to Makefile](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/c794e7293acdce9878e63a36f14da95de3cad564/Makefile#L7).

Reply:
it doesnt make a difference of course, it is just semantic that podman is the preferable suggestion please

Reply:
Done.